src/pandablocks_ioc/_pvi.py

```diff
@@ -295,9 +316,9 @@ def create_pvi_records(record_prefix: str):
             for group, components in v.items():
                 children.append(
                     Group(name=group.name, layout=Grid(), children=components)
-                )
+                )  # type: ignore
```

Comment:
I think if we change 

https://github.com/PandABlocks/PandABlocks-ioc/blob/6eadee5057eeae38fef75e2ba693d6d8ce17fe53/src/pandablocks_ioc/_pvi.py#L253

to use `ComponentUnion` (more accurate anyway), then some of the current mypy errors will go away.

If we change 

https://github.com/PandABlocks/PandABlocks-ioc/blob/e3472c22e5add39196b3bc3f2abefbfc0aeac702/src/pandablocks_ioc/_pvi.py#L311

to `children: List[ComponentUnion]` then we shouldn't need both type ignore's here.

If we can't do this then I think it'll be time to restrict `Tree` to a list on the pvi side.